### PR TITLE
St557

### PR DESCRIPTION
**Pull request by iEldor**

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -2,4 +2,7 @@
 friends = ['Bob', 'Ann']
 
 # added this line
-print('I was added as part of ST556 branch by another contributor')
\ No newline at end of file
+print('I was added as part of ST556 branch by another contributor')
+
+# added another line
+print('I was added as part of ST557 branch by another contributor')
```
Comment (skillrill) [inline test.py:8]: Add more details

**Review by skillrill — CHANGES_REQUESTED** [review]
add more details

**Comment by iEldor** [comment]
added details

File: test.py (hunk 1 above)
Comment (iEldor) [reply test.py:8]: added

**Review by skillrill — APPROVED** [review]

**Pull request merged**